cves/CVE-2016-6817.yml

```diff
@@ -130,7 +131,7 @@ subsystem:
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged.
     Examples: "clipboard", "gpu", "ssl", "speech", "renderer"
-  answer:
+  answer: http2
   name:
```

Comment:
what subsystem is this vulnerability a part of? Maybe Networking?

Reply:
Name added in new push